charts/warden/values.yaml

```diff
@@ -42,7 +42,7 @@ global:
       logging:
         format: json
         level: info
-
+  highPriorityClassName: "kyma-system-priority"
```

Comment:
Do we need to setup this here? It makes thath warden requires `cluster-essentials`